tests/common/fixtures/duthost_utils.py

```diff
@@ -3,17 +3,36 @@
 
 logger = logging.getLogger(__name__)
 
-@pytest.fixture
-def backup_and_restore_config_db(duthost):
-    """
-    Some cases will shutdown interfaces or BGP in test, and the change is writen to
-    config db after warm-reboot. Therefore, we need to backup config_db.json before
-    test starts and then restore after test ends
+
+def _backup_and_restore_config_db(duthost):
```

Comment:
@wangxin do you know of a better way of defining a fixture that can be used in module or function-level scope? I think having two fixtures could be confusing but I couldn't think of a better way of handling this situation.

Reply:
No, I don't think there is a better way than defining two fixtures.